README.md

```diff
@@ -4,6 +4,7 @@
 
 - **Version 2.1 (breaking changes from 2.0.x)**
 
+  - 2.1.28: Fixed PrimaryTopNav dropdown token and added new features for dropdown.
```

Comment:
I don't see any other features beside the tokens are these feateres not pushed yet?

Reply:
No, its not possible to change the text color when hovering on the dropdown